README.md

```diff
@@ -15,55 +15,110 @@ Clone this repo and `cd` into it.
+* ❓ In your own words, what is CORS?
+
+
+## Exploration Activity
 
 In small groups, designate one person to be the screen sharer.
 
-1. FE students, explain to the group what happens when you visit `localhost:3000` in your browser. Start with `index.html` and move to `index.js`. From there, trace all the way through the code up to the point where a request to the BE is made.
-1. BE, explain to the group what happens when the app receives the request. Trace all the way through the code up to the point that a response is sent back to the FE.
-1. FE, explain what happens once the FE receives the response.
-1. Once you are finished exploring the code, draw a diagram that illustrates the entire process.
+1. FE students: explain to the group what happens when you visit `localhost:3000` in your browser. Start with `index.html` and move to `index.js`. From there, trace all the way through the code up to the point where a request to the BE is made.
+1. BE students: explain to the group what happens when your Rails app receives a request. Trace all the way through the code up to the point that a response is sent back to the FE.
+1. FE students:, explain what happens once the FE app receives the response.
+1. Once you are finished exploring the code, draw a diagram that illustrates the entire process. Specify which parts are FE and which are BE. 
+
+
+
+# Extension Ideas
+
+
+### Create & Consume a New API Endpoint
+
+How would you collaborate as a combined team to consume a new set of data? In both applications, implement the ability to list *all* the messages from the BE's database on a new FE page. 
+
 
-**Extension**
+### More Exploration Questions
 
-If you have time, answer these questions
+If you have time, answer these questions together:
 
 * Is there any technical vocab words that the developers in the other program use that you don't understand? If so, try to help each other understand the meaning of those words.
 * How could you get the app to display one of the other messages found in `seeds.rb`?
-* What’s the difference between a database and an api? Is there an analogy that can help describe this?
-* What are frustrations that BE developers might experience when working with FE developers (for example, building an api/database based on specific requests from the FE team)?
-* What are frustrations that FE developers might experience when using technology built by BE developers (for example, accessing an api)?
+* What’s the difference between a database and an API? Is there an analogy that can help describe this?
+* What are frustrations that BE developers might experience when working with FE developers (for example, building an API/database based on specific requests from the FE team)?
+* What are frustrations that FE developers might experience when using technology built by BE developers (for example, accessing an API)?
+* What are some similarities between React and Rails? 
 
-## Checks for Understanding
+# Checks for Understanding
```

Comment:
Idea: Can we make this a Google quiz that we require they turn in for a little extra accountability? Perhaps we could review the quiz during the next moring's Daily Warmup?